src/uct/ib/rdmacm/rdmacm_md.c

```diff
@@ -72,6 +99,7 @@ int uct_rdmacm_is_sockaddr_accessible(uct_md_h md, const ucs_sock_addr_t *sockad
         /* Server side to check if can bind to the given sockaddr */
         if (rdma_bind_addr(cm_id, (struct sockaddr *)sockaddr->addr)) {
             ucs_debug("rdma_bind_addr() failed: %m");
+            uct_rdmacm_print_ip((struct sockaddr *)sockaddr->addr);
```

Comment:
need to write this as single message. for example:
`ucs_debug("rdma_bind_addr(addr=%s) failed: %m", ucs_sockaddr_str(sockaddr));`

Reply:
can you please suggest how this can be done without adding new parameters only for the error flow? or maybe we should keep the ip in the md anyway?

Reply:
which new parameter should be added?

Reply:
a parameter to hold the ip array.

Reply:
we could add a local var for err flow and pass it to ucs_sockaddr_str()